src/sb1-file.js

```diff
@@ -25,6 +28,22 @@ class SB1File {
         this.dataHeader.validate();
     }
 
+    get json () {
+        return toSb2Json({
+            info: this.info(),
+            stageData: this.data(),
+            images: this.images(),
+            sounds: this.sounds()
+        });
+    }
+
+    get zip () {
+        return toSb2FakeZipApi({
+            images: this.images(),
+            sounds: this.sounds()
```

Comment:
Should this also include the json?

Reply:
Ah, I see that this is called in VM and explicitly does not include the json. Maybe make a note of that here in a comment?